lib/urlwatch/handler.py

```diff
@@ -216,3 +221,41 @@ def parse_urls_txt(urls_txt):
 
     return jobs
 
+def parse_urls_yaml(urls_yaml):
+    import yaml
```

Comment:
No problem. Idea was to not introduce a new dependency, if the new method / config file type is not used.